packages/dds/tree/src/index.ts

```diff
@@ -159,6 +159,23 @@ export {
 	getJsonSchema,
 	type LazyItem,
 	type Unenforced,
+	type SimpleNodeSchemaBase,
+	type SimpleTreeSchema,
+	type SimpleNodeSchema,
+	type SimpleFieldSchema,
+	type SimpleLeafNodeSchema,
+	type SimpleMapNodeSchema,
+	type SimpleArrayNodeSchema,
+	type SimpleObjectNodeSchema,
+	normalizeFieldSchema,
+	isTreeNodeSchemaClass,
+	normalizeAllowedTypes,
+	getSimpleSchema,
+	numberSchema,
+	stringSchema,
+	booleanSchema,
+	handleSchema,
+	nullSchema,
```

Comment:
Are these needed? Can we not use the `schemaFactory.number`, etc. pattern? Not super concerned about exposing these as `@internal`, just wondering if it's strictly necessary.

Reply:
You can omit them but as a result I'd have to make a new instance of a SchemaFactory to access them. Having to create an instance of a factory seems a bit off but it's also not a big deal. What pattern do you recommend?

example code:
```
const sf = new SchemaFactory('test');
sf.number.identifier
``

Reply:
Yeah, that is awkward, and I've run into this myself. That said, for most use-cases, you can probably get away without comparing schema identifiers directly. I still need to do a more in-depth pass over the PR (I'll try to do that today), but what is the scenario where you need to compare identifiers? We do have various helpers for determining if something is a leaf vs map vs object, etc., and leaf schemas expose a `kind` field that will tell you if the leaf is a number vs string etc.